templates/bundles/DemosPlanCoreBundle/UI/form/element.html.twig

```diff
@@ -40,7 +40,7 @@
     small: ' u-1-of-4',
     smaller: ' u-1-of-5',
     tiny: ' u-1-of-6',
-    mini: ' u-1-of-8',
+    mini: ' u-1-of-2',
```

Comment:
Why do you apply this change? I do not see the need to do so within this PR, also, i guess it changes layout in many places where `element.html.twig` is used.

Reply:
the `Alles markieren` looks like this way 
![markerall](https://github.com/demos-europe/demosplan-core/assets/104975238/18ab8d10-9104-4692-a41b-3943ef871f29)

Reply:
done https://github.com/demos-europe/demosplan-core/pull/2832/commits/bfe56a2aee460e5ba39af47763135addad439511

Reply:
yes did it here https://github.com/demos-europe/demosplan-core/pull/2832/commits/bfe56a2aee460e5ba39af47763135addad439511

Reply:
i guess the `0` should not be a string but an actual number.

Reply:
I see <.< I was wondering, why it ditnt look like I expect it. Now it looks fine, thanks. done here https://github.com/demos-europe/demosplan-core/pull/2832/commits/10adbfac7a6ec2e91a3f0c7ef65f2f57277fed32